spec/features/team/view_team_members_spec.rb

```diff
@@ -34,16 +34,15 @@
     end
 
     context 'when there are many users in my organisation' do
-      before do
-        create(:user, email: 'friend@example.gov.uk', organisation: organisation)
-      end
+      let!(:user_2) { create(:user, email: 'bob@example.gov.uk', organisation: organisation) }
```

Comment:
can these be defined in numerical order please. (`[user_1, user_2]`)

It's jarring to read, and I can guarantee someone will "fix" it later and inadvertently break the test in a silent way